src/zenml/zen_stores/schemas/curated_visualization_schemas.py

```diff
@@ -0,0 +1,235 @@
+            resource_id=request.resource_id,
+            resource_type=request.resource_type.value,
+        )
+
+    def update(
+        self,
+        update: CuratedVisualizationUpdate,
+    ) -> "CuratedVisualizationSchema":
+        """Update a schema instance from an update model.
+
+        Args:
+            update: The update definition.
+
+        Returns:
+            The updated schema.
+        """
+        changes = update.model_dump(exclude_unset=True)
+        layout_size_update = changes.pop("layout_size", None)
+        if layout_size_update is not None:
+            self.layout_size = layout_size_update.value
+
+        for field, value in changes.items():
+            if hasattr(self, field):
+                setattr(self, field, value)
+
+        from zenml.utils.time_utils import utc_now
+
+        self.updated = utc_now()
+        return self
+
+    def to_model(
+        self,
+        include_metadata: bool = False,
+        include_resources: bool = False,
+        **kwargs: Any,
+    ) -> CuratedVisualizationResponse:
+        """Convert schema into response model.
+
+        Args:
+            include_metadata: Whether to include metadata in the response.
+            include_resources: Whether to include resources in the response.
+            **kwargs: Additional keyword arguments.
+
+        Returns:
+            The created response model.
+        """
+        layout_size_value = (
+            self.layout_size or CuratedVisualizationSize.FULL_WIDTH.value
+        )
+        try:
+            layout_size_enum = CuratedVisualizationSize(layout_size_value)
+        except ValueError:
+            layout_size_enum = CuratedVisualizationSize.FULL_WIDTH
+
+        resource_type_str = self.resource_type
+        if resource_type_str:
+            try:
+                resource_type_enum = VisualizationResourceTypes(
+                    resource_type_str
+                )
+            except ValueError:
+                resource_type_enum = VisualizationResourceTypes.PROJECT
+        else:
+            resource_type_enum = VisualizationResourceTypes.PROJECT
+
+        body = CuratedVisualizationResponseBody(
+            project_id=self.project_id,
+            created=self.created,
+            updated=self.updated,
+            artifact_version_id=self.artifact_version_id,
+            visualization_index=self.visualization_index,
+            display_name=self.display_name,
+            display_order=self.display_order,
+            layout_size=layout_size_enum,
+            resource_id=self.resource_id,
+            resource_type=resource_type_enum,
+        )
+
+        metadata = None
+        if include_metadata:
+            metadata = CuratedVisualizationResponseMetadata()
+
+        response_resources = None
+        if include_resources:
+            artifact_version_model = self.artifact_version.to_model(
+                include_metadata=include_metadata,
```

Comment:
This is not how we usually handle this. If some other model is included in the resources of a response, that sub-model will only have a body. Which means both `include_metadata` as well as `include_resources` should be `False` here.

Is there any reason we need to make this exception here?